lib/bolt/node.rb

```diff
@@ -53,15 +43,27 @@ def initialize(host, port = nil, user = nil, password = nil, uri: nil,
       @orch_task_environment = transport_conf[:orch_task_environment]
       @extensions = transport_conf[:extensions]
 
-      @logger = Logger.get_logger(progname: @host)
-      @transport_logger = Logger.get_logger(progname: @host, log_level: Logger::WARN)
+      @logger = Logger.get_logger(progname: host)
+      @transport_logger = Logger.get_logger(progname: host, log_level: Logger::WARN)
+    end
+
+    def host
```

Comment:
Do we use these enough to justify the top level methods for them? We'll have to translate `host
 -> target.host` and `port -> target.port` in the transports at some point anyway. I don't know if thats worth fixing now or later.